src/Components/Nav/index.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react"
+import NavLinks from "../NavLinks"
+
+function Nav() {
+
+   const [modalOpen, setModalOpen] = useState(false)
+
+   function handleClick() {
+    setModalOpen(!modalOpen)
+   }
+   return (
+    <div>
+        <button className='border px-3 rounded' onClick={handleClick}>{modalOpen ? '' : ''} MENU</button>
```

Comment:
What is this doing?

`{modalOpen ? '' : ''}`

If model is open, show '', otherwise show ''?